src/ReservoirPriceOracle.sol

```diff
@@ -141,11 +146,12 @@ contract ReservoirPriceOracle is IPriceOracle, IReservoirPriceOracle, Owned(msg.
     /// for priceCache[aTokenA][aTokenB] but instead the prices of its constituent simple routes will be written.
     /// @param aTokenA Address of one of the tokens for the price update. Does not have to be less than address of aTokenB
     /// @param aTokenB Address of one of the tokens for the price update. Does not have to be greater than address of aTokenA
-    /// @param aRewardRecipient The beneficiary of the reward. Must implement the receive function if is a smart contract address
-    function updatePrice(address aTokenA, address aTokenB, address aRewardRecipient) external nonReentrant {
+    /// @param aRewardRecipient The beneficiary of the reward. Must implement the receive function if is a smart
+    /// contract address. Set to address(0) if not seeking a reward
```

Comment:
Would change "must implement the receive function" to "must be able to receive ether" because im not sure if vyper calls it receive & I first thought this meant there was an interface to implement for receiving rewards.